website_v2/gcloud-powershell-website-v2/www/index.html

```diff
@@ -0,0 +1,70 @@
+﻿<!DOCTYPE html>
+<html class="google-js" lang="en" ng-app="powershellSite">
+<head>
+  <meta charset="utf-8">
+  <script>
+      (function(H){H.className=H.className.replace(/\bgoogle\b/,'google-js')})(document.documentElement)
```

Comment:
This was a carryover from the website that is currently published. Removing it doesn't seem to cause any issues- should I remove it or figure out who wrote it and ask them what it means?